lib/model/resources-config/ask-resources.js

```diff
@@ -8,6 +8,8 @@ const BASE = {
     profiles: {}
 };
 
+const FILE_NOT_FOUND_HINT_MESSAGE = ' If this is v1 skill, please run \'ask util upgrade-project\' then try the command again.';
```

Comment:
the message could be more clear if changing to "If this is a skill project managed by v1 ask-cli", v1 skill sounds like the very general v1 "Alexa skill"